### ISSUE TITLE
F1-score always be 0.0000e+00

### ISSUE THREAD

**Issue by Liu-Da** [tensorflow/addons#674]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): MacOS Mojave 10.14.5
- TensorFlow version and how it was installed (source or binary): '2.0.0' pip
- TensorFlow-Addons version and how it was installed (source or binary): '0.6.0' pip
- Python version: 3.7.0
- Is GPU used? (yes/no): yes

**Describe the bug**

I want to get the **F1-score** to measure the model after every epoch, but I always got 0.0000e+00 (F1-score).

**Code to reproduce the issue**
```
import tensorflow as tf
from tensorflow import keras
import tensorflow_addons as tfa

x = tf.random.uniform([400, 10])
y = keras.utils.to_categorical(tf.random.uniform([400, 1]) > 0.5)

model = tf.keras.models.Sequential([
  tf.keras.layers.Dense(100, activation='relu',input_shape=(10,)),
  tf.keras.layers.Dense(2, activation='softmax')
])

model.compile(optimizer=keras.optimizers.Adam(),
            loss="categorical_crossentropy" ,
            metrics=["Recall","Precision",tfa.metrics.F1Score(num_classes=3,average="micro")])

model.fit(x = x, y = y,
          epochs = 5,
          batch_size = 10,
          validation_split=0.1,
          callbacks=[keras.callbacks.EarlyStopping("val_f1_score", patience=3, restore_best_weights=True)])
```

**Other info / logs**

Train on 360 samples, validate on 40 samples
Epoch 1/5
360/360 [==============================] - 1s 3ms/sample - loss: 0.6863 - Recall: 0.5333 - Precision: 0.5333 - f1_score: 0.0000e+00 - val_loss: 0.7083 - val_Recall: 0.4750 - val_Precision: 0.4750 - val_f1_score: 0.0000e+00
Epoch 2/5
360/360 [==============================] - 0s 389us/sample - loss: 0.6812 - Recall: 0.5611 - Precision: 0.5611 - f1_score: 0.0000e+00 - val_loss: 0.7143 - val_Recall: 0.4500 - val_Precision: 0.4500 - val_f1_score: 0.0000e+00
Epoch 3/5
360/360 [==============================] - 0s 406us/sample - loss: 0.6794 - Recall: 0.5639 - Precision: 0.5639 - f1_score: 0.0000e+00 - val_loss: 0.7135 - val_Recall: 0.4000 - val_Precision: 0.4000 - val_f1_score: 0.0000e+00
Epoch 4/5
360/360 [==============================] - 0s 404us/sample - loss: 0.6756 - Recall: 0.6000 - Precision: 0.6000 - f1_score: 0.0000e+00 - val_loss: 0.7204 - val_Recall: 0.3750 - val_Precision: 0.3750 - val_f1_score: 0.0000e+00

**Comment by seanpmorgan** [comment]
cc @SSaishruthi code owner.


EDIT----
We also may want to verify this now that the refactor has been merged

**Comment by SSaishruthi** [comment]
@seanpmorgan 

Our new solution will solve this solve. We have added the threshold parameter for that purpose. I have checked with the new code and it works.

@Liu-Da Please use the new scripts to avoid this case

**Comment by seanpmorgan** [comment]
> @seanpmorgan
> 
> Our new solution will solve this solve. We have added the threshold parameter for that purpose. I have checked with the new code and it works.
> 
> @Liu-Da Please use the new scripts to avoid this case

Thanks @SSaishruthi !
